### PR TITLE
Atlas catalog integration

### PR DESCRIPTION
**Pull request by Colin-Stone**
Separate commits as I had to merge in Gateway script changes along the way

### CODE REVIEW

File: scripts/zowe-api-mediation-configure.sh [hunk 1]
```diff
@@ -139,6 +142,15 @@ services:
           serviceRelativeUrl: api/v1/datasets
         - gatewayUrl: ui/v1  # [api/ui/ws]/v{majorVersion}
           serviceRelativeUrl: ui/v1/datasets
+      apiInfo:
+        - apiId: com.ibm.datasets
+          gatewayUrl: api/v1
+          version: 1.0.0
```
Comment (stevenhorsman) [inline scripts/zowe-api-mediation-configure.sh]: Can you update the version to 0.9.3 (I know it doesn't match the v1 flag, but we know they will change before 1.0)

File: scripts/zowe-api-mediation-configure.sh [hunk 2]
```diff
@@ -178,6 +213,9 @@ cat <<EOF >$TEMP_DIR/orion.yml
 #
 services:
     - serviceId: orion
+      title: IBM z/OS Orion
+      description: IBM z/OS Orion Languages REST API service
+      catalogUiTileId: orion
```
Comment (stevenhorsman) [inline scripts/zowe-api-mediation-configure.sh]: This shouldn't have a catalog UI tile

Comment (Colin-Stone) [reply scripts/zowe-api-mediation-configure.sh]: It doesn't. It's the 
catalogUiTiles:
    zos:
        title: z/OS Miscellaneous services
        description: IBM z/OS Miscellaneous REST services

that does that. However the text is somewhat redundant though

File: scripts/zowe-api-mediation-configure.sh [hunk 3]
```diff
@@ -155,21 +170,69 @@ services:
...
+        description: IBM z/OS Jobs REST services
 EOF
 iconv -f IBM-1047 -t IBM-850 $TEMP_DIR/jobs.yml > $STATIC_DEF_CONFIG/jobs.yml	
 
+# Add static definition for uss
+cat <<EOF >$TEMP_DIR/uss.yml
+#
+services:
+    - serviceId: uss
+      title: IBM z/OS USS
```
Comment (stevenhorsman) [inline scripts/zowe-api-mediation-configure.sh]: Can you change the title/description to z/OS Unix Files as I think that'll be clearer to people

Comment (Colin-Stone) [reply scripts/zowe-api-mediation-configure.sh]: Makes sense

**Pull request merged**